x/src/cargo.rs

```diff
@@ -86,39 +107,65 @@ impl Cargo {
         self
     }
 
+    // Passes extra environment variables to be passed to the executable.
+    pub fn env_additions<I, S>(&mut self, envs: I) -> &mut Self
+    where
+        I: IntoIterator<Item = (S, S)>,
+        S: AsRef<OsStr>,
+        //        S: std::convert::AsRef<std::ffi::OsStr>,
+    {
```

Comment:
This still isn't the function signature I recommend you change this to. Please take a look at [Command::envs](https://doc.rust-lang.org/stable/std/process/struct.Command.html#method.envs)

Reply:
sorry in a world with inheritance  s,s would be fine, we're not in that world.  old habits.

Reply:
also was thinking you meant the type signature of the method, figure now you meant the entire sig, including the name.

Reply:
done.